### DISCUSSION TITLE
[BootstrapVue warn]: observeDom: Requires MutationObserver support.

### DISCUSSION THREAD

**Discussion by rodhoward** [bootstrap-vue/bootstrap-vue#5946] [Q&A]
### Describe the bug
We are unit testing our Vue components using jest and vue-test-utils.  And we are getting hundreds of warnings:
```
console.warn node_modules/bootstrap-vue/dist/bootstrap-vue.common.js:780
    [BootstrapVue warn]: observeDom: Requires MutationObserver support.
```

Our test code is something like:
```
import Vuex from "vuex";
import { mount, createLocalVue } from "@vue/test-utils";
import BootstrapVue from "bootstrap-vue";

// didn't help?
const mutationObserverMock = jest.fn(function MutationObserver(callback) {
  this.observe = jest.fn();
  this.disconnect = jest.fn();
  // Optionally add a trigger() method to manually trigger a change
  this.trigger = mockedMutationsList => {
    callback(mockedMutationsList, this);
  };
});
global.MutationObserver = mutationObserverMock;

const localVue = createLocalVue();
localVue.use(BootstrapVue);
localVue.use(Vuex);
...
...
wrapper = mount(myModalComponent, {
        store,
        localVue,
        propsData: { staticModal: true }
      });
```

### Steps to reproduce the bug

code similar to above and run your tests.

### Expected behavior

No scrolling through warning messages.

### Versions

**Libraries:**
"vue": "^2.5.21",
"@vue/test-utils": "^1.0.0-beta.32",
"bootstrap-vue": "^2.5.0",
"jest": "^26.6.0",

**Environment:**

- Device: Desktop Amd 
- OS: [e.g. Ubuntu ]
- Browser: [jest dom.. ]
- Version: [e.g. 70]

**Comment by rodolfo-santos** [comment]
Were you able to solve it?

**Reply by rodhoward** [reply]
Nope

**Reply by Mogge** [reply]
I did `global.process.env.BOOTSTRAP_VUE_NO_WARN = true` in my test setup file

**Comment by Mogge** [comment]
Any solutions?
I ran into the same problem testing `b-tabs`
